sunbeam-python/sunbeam/commands/openstack.py

```diff
@@ -272,7 +272,7 @@ def run(self, status: Optional[Status] = None) -> Result:
 
 
 class PatchLoadBalancerServicesStep(BaseStep):
-    SERVICES = ["traefik", "traefik-public", "rabbitmq", "ovn-relay"]
```

Comment:
This will most likely fail if the storage role is not enabled

Reply:
Fixed